README.md

```diff
@@ -2,4 +2,189 @@
+
+Create a custom class called `FISSentence` that inherits from `NSObject`.
+
+#### Properties
+
+In the `FISSentence.h` header file, declare a public `readonly` property that is an `NSString` called `sentence`.
+
+In the `FISSentence.m` implementation file, create a private `@interface` section between the `#import` lines and the `@implementation` section with the correct syntax:
+
+```objc
+@interface FISSentence ()
+
+@end
+```
+
+Within this private `@interface`:
+
+  *  redeclare the `sentence` property as `readwrite`,
+  *  declare a private `NSMutableArray` property called `clauses`, and
+  *  declare a private `NSMutableArray` property called `punctuations`.  
+  **Top-tip:** *Notice the trailing "s" in the property names. Even though "punctuation" is its own plural form, our code needs to disambiguate the collection's name from the name of its contents. Even though this is linguistically awkward, it's regarding among programmers as best practice to use the trailing "s" character for collection names despite the linguistic inaccuracy.*
+  
+#### Public Methods
+
+In the `FISSentence.h` header file, declare the eight (8) following public methods, none of which provide a return (i.e. are return-type `void`):
+
+  * `addClause:withPunctuation:` which takes two arguments, an `NSArray` called `clause` and an `NSString` called `punctuation`,
+
+  * `removeClauseAtIndex:` which takes one argument, an `NSUInteger` called `clausesIndex`,
+
+  * `insertClause:withPunctuation:atClausesIndex:` which takes three arguments, an `NSArray` called `clause`, an `NSString` called `punctuation`, and an `NSUInteger` called `clausesIndex`,
+
+  * `replacePunctuationForClauseAtIndex:withPunctuation:` which takes two arguments, an `NSUInteger` called `clausesIndex` and an `NSString` called `punctuation`,
+
+  * `addWord:toClauseAtIndex:` which takes two arguments, an `NSString` called `word` and an `NSUInteger` called `clausesIndex`,
+
+  * `removeWordAtIndex:fromClauseAtIndex:` which takes two arguments, an `NSUInteger` called `wordsIndex` and an `NSUInteger` called `clausesIndex`,
+
+  * `insertWord:atIndex:inClauseAtIndex:` which takes three arguments, an `NSString` called `word`, an `NSUInteger` called `wordIndex`, and an `NSUInteger` called `clauseIndex`, and
+
+  * `replaceWordAtIndex:inClauseAtIndex:withWord:` which takes three arguments, an `NSUInteger` called `wordsIndex`, an `NSUInteger` called `clausesIndex`, and an `NSString` called `word`.
+
+#### Private Methods
+
+##### Override `init`
+
+This lesson is not about writing initializers, however the private mutable array properties must be initialized when setting up the instance of `FISSentence`. This cannot be done from the outside of the class. Paste the following code into the `FISSentence.m`; you don't to understand it for this lab but it must be included. Keep the `init` method organized at the top of the `@implementation` section as you write out the other methods:
```

Comment:
Make those two array properties public and drop this section. Feel free to allude to coolness coming up